manifests/base/cuda-rstudio-imagestream.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-creator: RHOAI
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/rstudio"
+    opendatahub.io/notebook-image-name: "RStudio | Minimal | CUDA | R 4.5"
+    opendatahub.io/notebook-image-desc: "RStudio Server Workbench image with an integrated development environment for R, a programming language designed for statistical computing and graphics."
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+  name: cuda-rstudio-rhel9
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    # N Version of the image
+    - annotations:
+        # language=json
+        opendatahub.io/notebook-software: |
+          [
+            {"name":"CUDA","version":"12.8"},
+            {"name":"R","version":"v4.5"},
+            {"name":"Python","version":"v3.12"}
+          ]
+        # language=json
+        opendatahub.io/notebook-python-dependencies: |
+          [
+            {"name":"rstudio-server","version":"2025.09"}
+          ]
+        openshift.io/imported-from: quay.io/modh/cuda-rstudio
+        opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/default-image: "true"
```

Comment:
This should be removed. That annotation applies only on minimal notebook.
(The same for the rstudio-imagestream.yaml)